tests/ios/xcodeproj/Single-Static-Framework-Project.xcodeproj/bazelinstallers/lldb-settings.sh

```diff
@@ -11,13 +11,19 @@ set -euo pipefail
 # the paths in the debug info. This lldb setting ensures that _project relative_
 # paths are remapped to _project absolute_ paths.
 #
+# The platform settings one points the current working directory (cwd) to workspace root.
+# This is needed for features relying on lldb remote debugging, such as `oso_prefix_is_pwd`.
+#
 # NOTE: In order to use this, add this line to `~/.lldbinit`:
 #
 #     command source ~/.lldbinit-source-map
 cat <<-END > ~/.lldbinit-source-map
 settings set target.source-map ./ "$BAZEL_WORKSPACE_ROOT/"
 settings set target.sdk-path $SDKROOT
 settings set target.swift-framework-search-paths $FRAMEWORK_SEARCH_PATHS
+
+platform settings -w "$BAZEL_WORKSPACE_ROOT/"
```

Comment:
Interesting, I've only tested this by doing `script import os; os.chdir("$BAZEL_WORKSPACE_ROOT")` but this seems nicer if it works 🤔 

Do you have to reset this setting in other LLDB sessions? you do for my option

Reply:
@keith do you do `script import os; os.chdir("$BAZEL_WORKSPACE_ROOT")` in `~/.lldbinit-source-map` as well like here? It fails for me for some reason

Reply:
You have a good point since this is on user directory level. let me test this out!

Reply:
and this works btw, tested in downstream project that uses this

Reply:
IIRC with the script option lldb-rpc-server kept around some state about that change, where it looks like it doesn't with your change, which is awesome

Reply:
@keith so this is going to be a problem for bazel projects not built using this rules_ios because this script is embedded inside every xcodeproj generated here so running one will overwrite the content from the previous one.
As we are switching to use per scheme .lldbinit file in future, we will eventually pointing this script to a per-project file not user directory

Reply:
> IIRC with the script option lldb-rpc-server kept around some state about that change, where it looks like it doesn't with your change, which is awesome

Wanted to learn how u know about the state in this lldb-rpc-server, also i wonder what does `platform status` output looks like for you when use the script `os.chdir("$BAZEL_WORKSPACE_ROOT")`. This can be invoked in the debugging window with `lldb`